config/routes.rb

```diff
@@ -5,6 +5,6 @@
   root 'home#index'
 
   resources :books
-  resources :billings, only: :update
-  resources :shippings, only: :update
+  resources :users, only: :update
+  get '/settings', to: 'users#edit', as: 'settings'
```

Comment:
Please use REST routes, it should be   `resources :users, only: %w[edit update]`